vcd/resource_vcd_dnat.go

```diff
@@ -105,9 +113,11 @@ func resourceVcdDNATCreate(d *schema.ResourceData, meta interface{}) error {
 	providedNetworkName := d.Get("network_name")
 	if nil != providedNetworkName && "" != providedNetworkName.(string) {
 		orgVdcnetwork, err = getNetwork(d, vcdClient, providedNetworkName.(string))
-	} else {
-		_, _ = fmt.Fprint(GetTerraformStdout(), "WARNING: This resource will require network_name in the next major version \n")
 	}
+	// TODO enable when external network supported
+	/*else {
+		_, _ = fmt.Fprint(GetTerraformStdout(), "WARNING: This resource will require network_name in the next major version \n")
```

Comment:
Please add `network_type and network_name` to the message so we don't forget later.

Reply:
Changed